src/components/Items/ListItem.vue

```diff
@@ -0,0 +1,84 @@
+<template>
+  <q-item v-ripple clickable>
+    <q-item-section @click="loadList()">
+      <q-input
+        :autofocus="!isEdit"
+        :value="list.name"
+        @input="updateList({ id: list.id, updates: { name: list.name } })"
```

Comment:
This needs to be fixed. Currently I should save the new name only when the user save it